docs/introduction.rst

```diff
@@ -12,12 +12,23 @@ Dependencies
 Data levels
 ===========
 
-* R0
-* R1
-* DL1a
-* DL1b
-* DL2
-* DL3
++------+---------------------------------------------------------------------------+-------------+
+| Level| Description                                                               | File Format |
++======+===========================================================================+=============+
+| R0   | Uncalibrated RAW waveforms from the camera                                | ZFITS       |
++------+---------------------------------------------------------------------------+-------------+
+| R1   | Calibrated waveforms from the camera                                      |             |
++------+---------------------------------------------------------------------------+-------------+
+| DL1a | Integrated charge and peak position of the waveform                       | HDF5        |
++------+---------------------------------------------------------------------------+-------------+
+| DL1b | Image parameters (size, shape, intensity, etc.)                           | HDF5        |
```

Comment:
size is confusing because historically used for intensity. Better just "(width, length, intensity, etc.)"